src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -147,6 +295,22 @@ export default class FileField extends React.Component {
                 errorSchema,
               );
               const attachmentNameErrors = _.get([index, 'name'], errorSchema);
+              const showPasswordInput =
+                requestLockedPdfPassword && // feature flag
+                (files[index].isEncrypted || // needs password
+                  // incorrect password, returning error message
+                  (files[index].password && files[index].errorMessage));
+              const showPasswordSuccess =
+                requestLockedPdfPassword &&
+                file.password &&
+                !hasErrors &&
+                !file.errorMessage;
```

Comment:
If `file.errorMessage` was included in `errors`, then we wouldn't have to make this confusingly duplicate check